docs/development/Setup_Integrationtests_de.md

```diff
@@ -0,0 +1,32 @@
+# Setup für die Integrationstests
+
+In diesem Abschnitt werden die benötigten Schritte beschrieben um die Integrationstests für Portainer korrekt ausführen zu können.
```

Comment:
Copy-paste error, Integrationtests for Redmine

Reply:
Done :+1: